### PR TITLE
feat: rename "report a problem" field to "add a comment"

### PR DESCRIPTION
**Pull request by matthieu-foucault**

### CODE REVIEW

File: app/components/Forms/AddCommentField.tsx [hunk 1]
```diff
@@ -52,21 +58,23 @@ const ProblemReportField: React.FunctionComponent<FieldProps<string>> = (
...
           <p>
-            Please leave a detailed message below explaining any issues
-            encountered on this page.
-            <br />
-            This message will be sent with your application when it is
-            submitted.
-            <br />A representative from the Climate Action Secretariat may
-            contact you for further clarification.
+            For any technical issue, or to review documentation, please click on
+            the help bubble in the bottom right of your screen.
```
Comment (kriscooke) [inline app/components/Forms/AddCommentField.tsx:77]: Minor detail (what they'll see is a button):
```suggestion
            the help button in the bottom right of your screen.
```
I'm also wondering if we can delay adding this sentence and add it as part of the card implementing the help button (could cause confusion for QA testers)?

**Review by pbastia — APPROVED** [review]
Looks great!  👍 

The success message in the toast notification is a little hard for me to understand quickly - but it's probably my English that is at fault here

File: app/components/Forms/AddCommentField.tsx (hunk 1 above)
Comment (matthieu-foucault) [reply app/components/Forms/AddCommentField.tsx:77]: I would expect that help button to be in our next release to QA

Comment (kriscooke) [reply app/components/Forms/AddCommentField.tsx:77]: OK. Either way, "button" is more correct - they will not see any bubble in the bottom of the screen.

**Review by kriscooke — APPROVED** [review]

**Review by kriscooke — APPROVED** [review]

**Pull request merged**